docs/guide/content_rendering.md

```diff
@@ -6,7 +6,7 @@ eZ Platform comes with a native controller to display your content, known as the
 
 It can also be called straight by its direct URI: 
 
-`/view/content/<contentId>/full/true/<locationId>`
+`/view/content/<contentId>/full/1/<locationId>`
```

Comment:
is this maybe implying we have a bug, or are we sure it should be 0/1 ?

Reply:
NOTE:
current 2.5 version (ee demo with view.yml removed) behaves the same:
http://3.250.111.56:8001/view/content/221/full/0/223 => embeded in pagelayout
http://3.250.111.56:8001/view/content/221/full/false/223 => not embeded in pagelayout

Reply:
@alongosz WDYT?

Reply:
> @alongosz WDYT?

Well... It really depends on the expectation here. If we wanted to be able to use "true" or "false" then this is a bug. But maybe we didn't?
It's one of those "who needs strict types in PHP" confusion, though here we deal with array, so nothing on language level to be done (yet).
https://github.com/ezsystems/ezpublish-kernel/blob/v7.5.7/eZ/Publish/Core/MVC/Symfony/View/ParametersInjector/NoLayout.php#L27

`(bool)'false'` is actually `true`

Let's ask @bdunogier what he meant (sorry, git blame does not forget nor forgive ;) )

Side: default/fallback value in the linked line is inconsistent with route configuration, for which the fix is still pending via ezsystems/ezpublish-kernel/pull/2656. I'm actually not sure there if this is BC break or not.

Reply:
No need for forgiveness, it was a while ago ;)

If possible, I'd try to support both `0`/`1` and `false`/`true`. The parameter is called `layout`, so yes/no on it is fairly straightforward.

Or am I missing something ?